modules/javafx.fxml/src/main/java/javafx/fxml/FXMLLoader.java

```diff
@@ -153,7 +153,7 @@ public boolean isCollection() {
         }
 
         @SuppressWarnings("unchecked")
-        public void add(Object element) throws LoadException {
+        public void add(Object element) {
```

Comment:
by removing a checked exception we are changing public API.  do we really want to do that?

Reply:
This is part of a private class (`class Element`) that's nested within `FXMLLoader`, not public API.

Reply:
you are right, please disregard.